tests/e2e/notifications/batch/test_batches.py

```
@@ -0,0 +1,36 @@
+    assert len(result) == 1
+    assert result[0].id == batch_id
+
+
+@pytest.mark.skip(reason="Batches can not be deleted")
+def test_create_batch_with_file(batch_service, batch_data, logo_fd):
+    result = batch_service.create(batch_data, file=logo_fd)
+
+    assert result is not None
+
+
+@pytest.mark.skip(reason="Batches attachments not implemented")  # noqa: AAA01
```

Ruff (0.14.8)

Unused `noqa` directive (unknown: `AAA01`)

Remove unused `noqa` directive

(RUF100)